src/projects/automation/activities/activity.md

```diff
@@ -20,18 +20,20 @@ The `actionStatus` chip can have one of the following values:
 
 Since execution stops when an action fails, you'll most commonly want to use the `actionStatus` chip to check for `ContinuedWithErrors` or `SuccessWithTimeout` values (or the inverse: `actionStatus != "Success"`). These values indicate that the workflow continued but that something went wrong and may require corrective logic.
```

Comment:
```suggestion
An activity will stop executing if an action enters the `Failed` state and is not handled by a parent action.
To prevent the activity execution stopping, a user is able to use a combination of `Fail on Error` and `Error on Timeout` flags in the actions, in addition to the `actionStatus` datachip, to handle situations in a case by case basis. By using this approach, a user is able to implement corrective logic, in conjunction with the value of the `actionStatus` datachip, or gracefully clean down the activity before finishing execution.
```